src/components/grid-view/GridView.scss

```diff
@@ -37,17 +73,41 @@
 
 .bdl-GridView-slot {
     border: 1px solid $bdl-gray-10;
+    box-shadow: 0 1px 5px rgba(50, 50, 50, .1);
+    cursor: pointer;
     flex: 0 1 auto;
-    margin-top: 10px;
+    margin-bottom: 5px;
+    margin-top: 5px;
     overflow: hidden;
-    padding: 15px;
     white-space: normal;
 
+    &:hover,
+    &:active,
+    &:focus {
+        border-color: $bdl-box-blue-50;
+    }
+
+    &.bdl-GridView-slot--selected {
+        border-color: $bdl-box-blue-50;
+    }
+
+    &:first-child {
+        margin-left: 2px;
+    }
+
     & + & {
         margin-left: 10px;
     }
 }
 
 .bdl-GridView-tableRow {
     border: none;
+
+    &:first-child .bdl-GridView-slot {
```

Comment:
This can probably be simplified with sibling selectors:

```
.bdl-GridView-tableRow {
    & + & {
        margin-top: 10px;
    }
}
```

Reply:
Still looking into this.  React Virtualized seems to place the rows improperly if the margins aren't on the GridViewSlots.